ajax/libs/react-is/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "react-is",
+  "version": "16.12.0",
```

Comment:
Please don't include the version attr. The auto-update bot will populate this once it pulls in versioned files.

Reply:
I have removed the version